UnitTestFrameworkPkg/Include/Library/HostMemoryAllocationBelowAddressLib.h

```diff
@@ -0,0 +1,93 @@
+/** @file
+  HostMemoryAllocationBelowAddressLib class
+
+  Copyright (c) 2024, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#ifndef HOST_MEMORY_ALLOCATION_BELOW_ADDRESS_LIB_H_
+
+/**
+  Allocate memory below a specifies address.
+
+  @param[in] MaximumAddress  The address below which the memory allocation must
+                             be performed.
+  @param[in] Length          The size, in bytes, of the memory allocation.
+
+  @retval  !NULL  Pointer to the allocated memory.
+  @retval  NULL   The memory allocation failed.
+**/
+VOID *
+EFIAPI
+HostAllocatePoolBelowAddress (
+  IN UINT64  MaximumAddress,
+  IN UINT64  Length
+  );
+
+/**
+  Free memory allocated with AllocateMemoryHostAllocatePoolBelowAddress().
+
+  @param[in] Address  Pointer to buffer previously allocated with
+                      HostAllocatePoolBelowAddress().
+
+  @retval  TRUE   The buffer was freed.
+  @retval  FALSE  The buffer could not be freed..

```

Comment:
The function description indicates a `BOOLEAN` return type but the return type is `VOID`.

Reply:
Thanks.  I will fix.